aws/data_source_aws_ecr_authorization_token_test.go

```diff
@@ -0,0 +1,60 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccAWSEcrAuthorizationTokenDataSource_basic(t *testing.T) {
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "data.aws_ecr_authorization_token.repo"
```

Comment:
Nit: to match value stored, should be `dataSourceName`

Reply:
fixed in 6952792ee1f5ec2822554c923836b2d6ac9ae02e